server/src/main/java/org/elasticsearch/search/aggregations/support/MultiValuesSourceFieldConfig.java

```diff
@@ -62,16 +63,16 @@
         if (timezoneAware) {
             parser.declareField(MultiValuesSourceFieldConfig.Builder::setTimeZone, p -> {
                 if (p.currentToken() == XContentParser.Token.VALUE_STRING) {
-                    return DateTimeZone.forID(p.text());
+                    return ZoneId.of(p.text());
```

Comment:
Related to the version checks, should we support short IDs?  If not we should probably make a note in the breaking changes.